basic/exercise2/script2.js

```diff
@@ -1,10 +1,23 @@
 'use strict';
 
-var side1 = 5;
-var side2 = 6;
-var side3 = 7
-var trianglePerimeter = (side1 + side2 + side3) / 2;
-var triangleArea = Math.sqrt(trianglePerimeter * ((trianglePerimeter - side1) * (trianglePerimeter - side2) * (trianglePerimeter - side3)));
-document.getElementById("triangle").innerHTML = triangleArea;
+function findingTriangleArea(side1, side2, side3) {
```

Comment:
Please stop using continuous form in function name - it should not finish with ing. triangleArea is good enouth.